_resources/port1.0/group/cmake-1.1.tcl

```diff
@@ -57,6 +57,23 @@ default cmake.build_dir             {${workpath}/build}
 # minimal/initial value for the install rpath:
 default cmake.install_rpath         ${prefix}/lib
 
+# CMake provides several different generators corresponding to different utities
+# (and IDEs) used for building the sources. We support "Unix Makefiles" (the default)
+# and Ninja, a leaner-and-meaner alternative.
+#
+# In the Portfile, use
+#
+# cmake.generator Ninja
+# or
+# cmake.generator Unix Makefiles    # no quotes!
```

Comment:
There are a number of places (constructs) where Tcl introduces curly braces for reasons I don't fathom. I think that it must have something to do with differences between assigning to a variable directly (with `set`) and through a procedure call.
This is the way I've managed to get the feature to work; I need the quotes and curly braces in the default statement but the user-callable version can and must do without (fortunately).  Either way I'm receiving the value as `"{Unix Makefiles}"` in the option_proc.

If you know how to manage without I'd be happy to see how!